pandas/tests/test_window.py

```diff
@@ -533,6 +512,14 @@ def test_multi_index_names(self):
         tm.assert_index_equal(result.columns, df.columns)
         assert result.index.names == [None, '1', '2']
 
+    @pytest.mark.parametrize('cls', [pd.Series, pd.DataFrame])
```

Comment:
Can you use "klass" instead of "cls"

Reply:
I strongly prefer `cls` in this case. Having grown up in Georgia, certain misspellings have a distasteful cultural undertone for me. Also, `cls` is an equally popular standard in other Python projects, if not more popular.

Reply:
"klass" is most used in pandas so we should stick with that as the standard for now (thought admittedly not 100%). If you feel so inclined you can definitely bring up the conversation to standard as an issue or on the mailing list and get feedback. 

Would be easier to change in one sweep that having each contributor implement their own standard

```bash
grep "parametrize.*klass" -r pandas/tests/ | wc -l
      38
grep "parametrize.*box" -r pandas/tests/ | wc -l
      12
grep "parametrize.*cls" -r pandas/tests/ | wc -l
      10
```

Reply:
Ok, I'll make that change separately.